examples/certs/gcp/gcp.key

```diff
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
```

Comment:
I assume this was regenerated using the instructions in https://github.com/project-oak/oak/blob/main/examples/certs/README.md

@conradgrobler is there any reason why that is not an actual executable shell script with comments? also from what I can tell the process is still pretty interactive, i.e. it asks for country code etc., hopefully that can be scripted too?

Reply:
There is not a good reason. At the time I thought it would not be something we had to rerun often so went with just documenting the steps and leaving it as an interactive process.

The other fields can be specified in the config file that is passed to `openssl req` for generating the CSR (e.g. https://gist.github.com/yidas/af42d2952d85c0951c1722fcd68716c6)